app/views/fines/index.html.erb

```diff
@@ -9,33 +9,40 @@
   <h2>Payable: <%= number_to_currency(@fines.sum(&:owed)) %></h2>
 
   <% if @fines.any? && !params[:payment_pending] %>
-      <% unless params[:group] && patron.sponsor? %>
-        <div class="mb-3">
-          <%= render 'fines/pay_all_button' %>
-        </div>
-      <% else %>
+      <% if params[:group] && patron.sponsor? # temporary fix until Symphony CGI scripts are updated %>
         <div class="pb-3">
           Fines can be paid in My Library Account only by the borrower who accrued them. <br />
           To pay group fines using iJournal,
           <%= link_to contact_path, data: { 'mylibrary-modal' => 'trigger' } do %>
               contact Circulation & Privileges.
           <% end %>
         </div>
-      <% end %>
-      <div class="d-none d-md-flex row font-weight-bold list-header">
-        <div class="row col-md-4">
-          <div class="col-md-12 col-lg-6">Reason</div>
-          <div class="col-md-12 col-lg-6">Amount</div>
+      <% elsif params[:group] %>
+        <div class="pb-3">
+          <%= sul_icon :'sharp-error-24px', classes: 'text-recalled' %> The research group has unpaid fines that may affect the status of all proxies.
         </div>
-        <div class="col-md-5">Title</div>
-        <div class="row col-md-3">
-          <div class="col-md-12 col-lg-6">Author</div>
-          <div class="col-md-12 col-lg-6 call_number">Call number</div>
+      <% else %>
+        <div class="mb-3">
+          <%= render 'fines/pay_all_button' %>
         </div>
-      </div>
-      <ul class="fines list-group">
-        <%= render @fines %>
-      </ul>
+      <% end %>
+
+      <% unless params[:group] && !patron.sponsor? %>
```

Comment:
```suggestion
      <% if !params[:group] || patron.sponsor? %>
```

This might be  a little clearer? Or, maybe:

```
      <% if patron_or_group.is_a?(Patron) || patron.sponsor? %>
```